src/controllers/browser.ts

```diff
@@ -49,44 +47,17 @@ export async function initWhatsapp(
   if (!waPage) {
     return false
   }
+
+  whatsappCacheManagement.setup(waPage, options)
```

Comment:
Colocar dentro do trycatch

```suggestion
  whatsappCacheManagement.setup(waPage, options)
```